repository-data/webfiles/src/main/resources/site/scss/components/_button.scss

```diff
@@ -26,3 +26,8 @@
 
     @include core-font(16);
 }
+
+.button--small {
+    font-size: 12px;
```

Comment:
What do you think to button--tiny?

Reply:
`.button--tiny` or .`button--extra-small` works for me, just make it separate from .button-small, as it's already being used (see an example in the cookie banner component).

A quick question about the font size - is this button always going to have 12px font and responsive line-height? I haven't checked it but it might be worth keeping an eye on it for consistency.